torch/nn/functional.py

```diff
@@ -1116,8 +1119,7 @@ def adaptive_max_pool2d_with_indices(
     input: Tensor, output_size: BroadcastingList2[int],
     return_indices: bool = False
 ) -> Tuple[Tensor, Tensor]:
-    r"""
-    adaptive_max_pool2d(input, output_size, return_indices=False)
+    r"""adaptive_max_pool2d(input, output_size, return_indices=False)
```

Comment:
You can add a comment `# noqa D400`:
```suggestion
    r"""adaptive_max_pool2d(input, output_size, return_indices=False) # noqa D400
```

Reply:
I have added #noqa: D400 at appropriate places. Have run the code to check any errors (no docstring errors). Please review.
@svekars @mikaylagawarecki

Reply:
I don't think this: ` -> Tuple[Tensor, Tensor]: #noqa: D400` the correct way to apply that fix - you only apply to docstrings that have a function signature in them (like in the example above). pydocstyle errors because it wants a period at the end of that docstring. When you say # noqa D400 - you say "ignore the error that requires a period at the end of that sentence". You don't need to put it next to the missing docstrings errors - those errors as we said before can stay.

Reply:
I am a bit confused.
For clarification, I have not used noqa for missing docstring errors. I have used it only where we have function signatures that need to be there as pointed out by @mikaylagawarecki.
I did try to fix it by adding #noqa D400 inside docstring (as suggested above) - however, that did not resolve the D400 errors. On inspecting further, adding noqa after function ignores D400 for that function docstring. 
Current local output:
```
/content/pytorch/torch/nn/functional.py:3712 in public function `upsample` (skipping F811,B950):
        D103: Missing docstring in public function
/content/pytorch/torch/nn/functional.py:3717 in public function `upsample` (skipping F811,B950):
        D103: Missing docstring in public function
/content/pytorch/torch/nn/functional.py:3801 in public function `interpolate` (skipping F811,B950):
        D103: Missing docstring in public function
/content/pytorch/torch/nn/functional.py:3806 in public function `interpolate` (skipping F811,B950):
        D103: Missing docstring in public function
/content/pytorch/torch/nn/functional.py:3811 in public function `interpolate` (skipping F811,B950):
        D103: Missing docstring in public function
/content/pytorch/torch/nn/functional.py:3816 in public function `interpolate` (skipping F811):
        D103: Missing docstring in public function
/content/pytorch/torch/nn/functional.py:4071 in public function `upsample_nearest` (skipping F811):
        D103: Missing docstring in public function
/content/pytorch/torch/nn/functional.py:4076 in public function `upsample_nearest` (skipping F811):
        D103: Missing docstring in public function
/content/pytorch/torch/nn/functional.py:4109 in public function `upsample_bilinear` (skipping F811):
        D103: Missing docstring in public function
/content/pytorch/torch/nn/functional.py:4116 in public function `upsample_bilinear` (skipping F811):
        D103: Missing docstring in public function
/content/pytorch/torch/nn/functional.py:4123 in public function `upsample_bilinear` (skipping F811):
        D103: Missing docstring in public function
/content/pytorch/torch/nn/functional.py:4130 in public function `upsample_bilinear` (skipping F811):
        D103: Missing docstring in public function
/content/pytorch/torch/nn/functional.py:4745 in public function `assert_int_or_pair`:
        D103: Missing docstring in public function
13
```

Let me know if this is technically correct or not?